### DISCUSSION TITLE
ABP Framework on ASP.NET Community Standup!

### DISCUSSION THREAD

**Discussion by hikalkan** [abpframework/abp#10219] [Show and tell]
@hikalkan is presenting the ABP Framework on the .NET Community Standup. He is also explaining some coming features with ABP 5.0!

https://www.youtube.com/watch?v=vMWM-_ihjwM

**Comment by amsaighi** [comment]
I watched it live :D

**Reply by hikalkan** [reply]
Thanks :)
